Components/UploadData/index.js

```diff
@@ -5,7 +5,12 @@ import Container from "../Container";
 import FileUploader from "../FileUploader";
 import InputTypeSubmit from "../InputTypeSubmit";
 
-export default function UploadData({ fileObj, onUploadFile, onConversion }) {
+export default function UploadData({
+  fileObj,
```

Comment:
The prop would be more readable, when you don't cut it short

Reply:
We have renamed "fileObj" as "fileObject".